sql/aiven_extras.sql

```diff
@@ -453,6 +453,43 @@ END;
 $$;
 
 
+DROP FUNCTION IF EXISTS aiven_extras.set_pgaudit_param(TEXT, TEXT, anyelement);
+CREATE FUNCTION aiven_extras.set_pgaudit_param(
+    arg_parameter TEXT,
+    arg_database TEXT,
```

Comment:
Can we make this a bit easier by not specifying arg_database and applying this to current_database? That would mean you can only change stuff on the DB which you are allowed to be on and the superuser check is just another measure against missuse?

Reply:
Discussed it more and this seems good to go in